app/models/work.rb

```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
+
+class Work < ApplicationRecord
+  validates :title, presence: true
+
+  has_many :votes
+  has_many :users, through: :votes
+
+  def self.top_ten(media)
+    results = Work.where(media: media.to_s)
+    results = results.sort_by {|result| result.votes.length}.reverse
+
+    results = Work.where(media: media.to_s) if results.empty?
```

Comment:
What does this line do? Do we need it? Commenting it out still lets the tests pass